src/v2/models/AppState.js

```diff
@@ -123,10 +123,20 @@ export default Model.extend({
     }
 
     // default terminal state for fall back
+    //TODO: This is a FailSafe, it needs to be removed later by this Jira: OKTA-300044
     if (idxResp.context.terminal && _.isEmpty(idxResp.context.terminal.value)) {
       idxResp.terminal = {
         name: 'terminal',
-        value: [],
+        value: idxResp.context.messages.value.length ? idxResp.context.messages.value : [],
```

Comment:
how would you be able to read `context.messages` when `idxResp.context.terminal` is present?

Reply:
oops. forget to update that block.